Dockerfiles/midstream/run_tests.py

```diff
@@ -6,42 +6,47 @@
 import subprocess
 from os import path
 
+# global variables set 
+OPERATOR_DIR="/tmp/operator_dir/"
+OPERATOR_WORK_DIR="/tmp/test_operator_work_dir/"
+WORK_DIR="/tmp/output/"
+PLAYBOOKS_DIR="/tmp/operator-test-playbooks/"
+UMOCI_BIN_PATH="/usr/local/bin/umoci"
```

Comment:
this is already default value in the playbook.
```
extract-operator-bundle.yml:15:    umoci_bin_path: "/usr/local/bin/umoci"
```

Removing unnecessary parameter will make code simpler